src/OpenTelemetry/CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 ## Unreleased
 
+* Removed support for ParseStateValues if the state neither implements
+  IReadOnlyList nor IEnumberable when `ParseStateValues` is true.
+  ([#4560](https://github.com/open-telemetry/opentelemetry-dotnet/pull/4560))
+  > **Note**
```

Comment:
I would like to use 6/20 SIG meeting to get some clarity on:
1. What is the definition of a breaking change? Do we care about breaking alpha, beta or rc releases, or we only care about breaking stable release, i.e., public API changes as opposed to previous stable release.
2. Could we get a unified format when communicating breaking change to the users? For example:
    - Use the bolded text **Breaking change** in CHANGELOG (https://github.com/open-telemetry/opentelemetry-dotnet/pull/4542/files), and
    - Document which release and/or PR first introduced the change that will be breaking.

Reply:
In the SIG meeting, we agreed in using the format **Breaking Change** in CHANGELOG to communicate features shipped in the previous stable version stopped being supported.